app/components/submission.js

```diff
@@ -0,0 +1,172 @@
+            if (submission[key]?.date && submission[key]?.sent) {
+                return true
+            }
+        }
+        return false
+    }
+
+    const getSubmissionStatus = () => {
+        if (submission && !checkSubmissionBeforeDeadline()) return 'missed'
+        if (submission && checkVideoSubmitted()) return 'submitted'
+        if (checkReminderSent()) return 'sent'
+        return 'default'
+    }
+
+    const getSubmissionDaysLeft = () => {
+        const dueDate = Date.parse(electionObj.timeline.moderatorSubmissionDeadline[0].date)
+        const currDate = new Date()
+        return Math.round((dueDate - currDate) / 86400000)
+    }
+
+    const getSubmissionDaysAgo = () => {
+        const sentDate = Date.parse(submission.date)
+        const currDate = Date.now()
+        console.log(currDate, sentDate)
+        return Math.round((currDate - sentDate) / 86400000)
+    }
+
+    let statusTitle
+    let statusDesc
+    let prevIcon
+    let cornerPic
+    let missed = false
+    switch (getSubmissionStatus()) {
+        case 'missed':
+            statusTitle = 'Deadline Missed!'
+            prevIcon = <SvgDeadlineMissed />
+            cornerPic = <SvgFeelingBlue />
+            missed = true
+            break
+        case 'submitted':
+            statusTitle = 'Video Submitted!'
+            statusDesc = getSubmissionDaysAgo() + ' days ago'
+            prevIcon = <SvgCheck />
+            break
+        case 'sent':
+            statusTitle = 'Reminder Sent!'
+            statusDesc = getSubmissionDaysLeft() + ' days left'
+            prevIcon = <SvgReminder />
+            break
+        case 'default':
+            statusDesc = getSubmissionDaysLeft() + ' days left'
+            statusTitle = 'No Submission Yet'
+            break
+    }
+
+    const icon = <div className={classes.icon}>{prevIcon}</div>
+    return (
+        <div className={cx(className, classes.container)} style={style}>
+            <div className={cx(classes.card, { [classes.backgroundRed]: missed })}>
+                <div className={classes.preview}>
+                    {submission?.url && getSubmissionStatus() === 'submitted' ? (
+                        <iframe src={submission?.url} frameborder='0'>
+                            {icon}
+                        </iframe>
+                    ) : (
+                        <div className={classes.placeholder}>{icon}</div>
+                    )}
+                </div>
+                <div className={classes.meta}>
+                    <p className={cx(classes.title, { [classes.colorWhite]: missed })}>{statusTitle}</p>
+                    <p className={cx(classes.desc, { [classes.colorLightRed]: missed })}>{statusDesc}</p>
+                </div>
+            </div>
+            <div className={classes.cornerPic}>{cornerPic}</div>
+        </div>
+    )
+}
+
+const useStyles = createUseStyles(theme => ({
+    container: {
+        padding: '2rem',
+        backgroundColor: '#ececec',
```

Comment:
I see this color in theme.js  backgroundColorApp
Since you are making changes, it's a good time to try the theme feature of react-css.
See app/components/candidate-table-input.js for an example using theme.js
I'm new to using themes too but I like having all these color definitions in one place.